src/ServiceControl.Infrastructure.RavenDB/Extensions.cs

```diff
@@ -16,5 +19,21 @@ public static void Query<TState>(this DocumentDatabase db, string index, IndexQu
                 onItem(doc, state);
             }
         }
+
+        public static void ThrowWhenIndexErrors(this EmbeddableDocumentStore documentStore)
+        {
+            var statistics = documentStore.DatabaseCommands.GetStatistics();
+
+            if (statistics.Errors.Length > 0)
+            {
+                var text = new StringBuilder();
+                text.AppendLine("Detected RavenDB index errors, please start maintenance mode and resolve the following issues:");
```

Comment:
I know we normally don't do it but I wonder if this would be a good case to actually have a link to the documentation in the text

Reply:
We do have one in the custom check message, so there is already precedent. I think we just link to the ServiceControl troubleshooting page which feels relatively safe to do.